src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -771,7 +771,7 @@ public function testScalarInSequence()
      * @see http://yaml.org/spec/1.2/spec.html#id2759572
      * @see http://yaml.org/spec/1.1/#id932806
      */
-    public function testMappingDuplicateKeyBlock()
+    public function testLegacyMappingDuplicateKeyBlock()
```

Comment:
missing `@group legacy` annotation (and no need to rename the test case then)
same below